code/web/src/modules/user/tests/Profile.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Profile from '../Profile.js';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import reducer from '../api/state.js';
+
+describe('Profile', () => {
+    const store = createStore(reducer)
+
+
+    it.skip('should render the Profile component',() => {
+        //setup
+        const mockLogout = jest.fn()
+        
+        let props={
+            user: {
+                details: {
+                    name: 'Bob Forehead',
+                    email: 'Khaleta@thisisfake.com'
+                }
+            }
+        }
+
+        render(<Provider store={ store }><Profile user={props.user} logout={mockLogout} /></Provider>)
```

Comment:
I have tossed a line out to see if we can get some clarity on unit testing redux components. And I am going to see what other resources we can find...